source/reporting/index.html.md.erb

```diff
@@ -145,6 +145,8 @@ You can also sign in to the [GOV.UK Pay admin tool](https://selfservice.payments
 
 See the <a href="https://govukpay-api-browser.cloudapps.digital/#searchpayments" target="blank">GOV.UK Pay API browser</a> for more details
 
+You must not search payments to check a payment's status during [payment flow](/payment_flow). Search results are asynchronous so the payment status may not be up-to-date.
```

Comment:
I would not say "search payments" but rather "use the search payments endpoint".. don't know if this is consistent with other parts of the documentation though.

```suggestion
You must not use the search payments endpoint to check a payment's status during [payment flow](/payment_flow). Search results are asynchronous so the payment status may not be up-to-date.
```

Reply:
Ah good point. The style would be "search payments using the API" - that will also make it consistent with https://github.com/alphagov/pay-tech-docs/pull/398/files so thanks for spotting!